cli/package.json

```diff
@@ -20,10 +20,11 @@
     "react-native",
     "react"
   ],
+  "homepage": "https://microsoft.github.io/code-push",
   "author": "Microsoft Corporation",
   "license": "MIT",
   "dependencies": {
-    "code-push": "1.0.0-beta",
+    "code-push": "^1.1.0-beta",
```

Comment:
Reading the [semver documentation](https://github.com/npm/node-semver#caret-ranges-123-025-004) confirms that the `1.1.1-beta` version would not even be allowed for dependencies specified with `^1.1.0-beta`. `1.1.0-beta.2` would be allowed though.